Doc/library/trace.rst

```diff
@@ -13,6 +13,10 @@ annotated statement coverage listings, print caller/callee relationships and
 list functions executed during a program run.  It can be used in another program
 or from the command line.
 
+.. seealso:: `Coverage.py <https://coverage.readthedocs.io/>`_, which is one
```

Comment:
Thanks @Carreau. I also changed the markup to [reST list](http://www.sphinx-doc.org/en/stable/markup/para.html#directive-seealso).